src/main/java/com/connexta/search/index/IndexManagerImpl.java

```diff
@@ -30,11 +34,14 @@ public IndexManagerImpl(@NotNull final CrudRepository crudRepository) {
   public void index(
       @NotBlank final String productId,
       @NotBlank final String mediaType,
-      @NotNull final InputStream inputStream)
+      @NotNull final MultipartFile file)
       throws IndexException {
-    // TODO verify Media Type for CST
     // TODO check that the product exists in S3
 
+    if (!mediaType.equals(MediaType.APPLICATION_JSON_VALUE)) {
+      throw new IndexException("The CST file's media type is not application/json");
```

Comment:
✏️ better wording?

Reply:
What you have now looks good. Maybe this is better:
```java
"Expected the content type to be " + MediaType.APPLICATION_JSON_VALUE + " but was " + mediaType
```